src/Storages/WindowView/StorageWindowView.cpp

```diff
@@ -927,6 +932,76 @@ void StorageWindowView::threadFuncFireEvent()
     }
 }
 
+Pipe StorageWindowView::read(
```

Comment:
btw there is a setting `stream_like_engine_allow_direct_select`, let's also support it.

Reply:
Indeed